### ISSUE TITLE
Line numbers overlap when response length > 99 lines long

### ISSUE THREAD

**Issue by sfertman** [Huachao/vscode-restclient#58]
Great plugin guys! I use it everyday for work.  
**TL;DR**:
- VSCode Version: 1.8.1
- OS Version: Ubuntu 16.04
- REST Client Version: 0.11.4

Steps to Reproduce:
- Run a request that returns a response that is longer than 99 lines
---
More details
--
When REST-client receives a response with more than 99 lines, some of the numbers are rendered one on top of the other (see screenshot)
![screenshot from 2017-01-09 11-26-09](https://cloud.githubusercontent.com/assets/17301942/21775570/ab358328-d664-11e6-82b6-a2230764fcd4.png)

I also noticed that rendering a long request (~2500 lines in my case) takes quite a bit if time (~1min) and when scrolling up/down to view, it is sluggish and laggy. This is similar to described in #56 in the sense that it is not a network issue (response time is ~0.5sec); however, response size is quite small compared with 10MB discussed there:
![image](https://cloud.githubusercontent.com/assets/17301942/21776155/2e287b30-d667-11e6-872c-e5cc9e4f8f12.png)
Let me know if you require more details.  
Thanks.

**Comment by Huachao** [comment]
@sfertman thanks for your usage. For the overlap issue, I have fixed in my master branch code, and will publish in next release ASAP. And for the performance issue, I have created a new issue #59 to track, and I will investigate how to improve the render performance, and will update status in that thread, and will notify you too. So I'd like to close this issue now.

**Comment by sfertman** [comment]
@Huachao, thanks for a great extension!

**Comment by Huachao** [comment]
@sfertman the bug has been fixed in latest version 0.12.0, you can have a try and verify it. 😄

**Comment by sfertman** [comment]
@Huachao. Verified. Line numbers work now. Thanks!
